javascript/code-challenges/challenges-03.test.js

```diff
@@ -0,0 +1,300 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function called addTwo that takes in an array and adds two to every value using a for loop. Place the new value in a new array. Return the new array.
+------------------------------------------------------------------------------------------------ */
+
+const addTwo = (arr) => {
+  // Solution code here...
+
+};
```

Comment:
- Really good good work on challenge 3 👍 
- For this one, can we just use a for loop and then for every value at a given index `arr[i]` add 2?
- We may need to declare some sort of variable since we have to return an arr with new values added in